_posts/2023-02-08-fsdsinet-models.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: FSD-SINet models released
+image: /assets/fsdsinet-models/activations.png
+category:
+- news
+- tensorflow
+---
+
+We are delighted to introduce the FSD-SINet models for sound event classification in Essentia.
+FSD-SINet is a family of CNN architectures trained in the [FSD50K](https://zenodo.org/record/4060432) dataset with techniques to enhance shift invariance.
```

Comment:
It would be nice to give easy access to the list of the classes here (a URL) because the link to Zenodo does not such list.

Reply:
We will show all the classes in the Essentia models' site. I mentioned this in the next paragraph.